cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -296,42 +189,24 @@ private ResourceOperatorSupplier supplierWithMocks() {
     }
 
-    private void assertResourceRequirements(VertxTestContext context, String statefulSetName) {
-        context.verify(() -> {
-            StatefulSet statefulSet = client.apps().statefulSets().inNamespace(NAMESPACE).withName(statefulSetName).get();
-            assertThat(statefulSet, is(notNullValue()));
-            ResourceRequirements requirements = statefulSet.getSpec().getTemplate().getSpec().getContainers().get(0).getResources();
-            if (resources != null && resources.getRequests() != null) {
-                assertThat(requirements.getRequests(), hasEntry("cpu", resources.getRequests().get("cpu")));
-                assertThat(requirements.getRequests(), hasEntry("memory", resources.getRequests().get("memory")));
-            }
-            if (resources != null && resources.getLimits() != null) {
-                assertThat(requirements.getLimits(), hasEntry("cpu", resources.getLimits().get("cpu")));
-                assertThat(requirements.getLimits(), hasEntry("memory", resources.getLimits().get("memory")));
-            }
-        });
-    }
 
     private Future<Void> initialReconcile(VertxTestContext context) {
         LOGGER.info("Reconciling initially -> create");
         return operator.reconcile(new Reconciliation("test-trigger", Kafka.RESOURCE_KIND, NAMESPACE, CLUSTER_NAME))
             .onComplete(context.succeeding(v -> context.verify(() -> {
-                StatefulSet kafkaSts = client.apps().statefulSets().inNamespace(NAMESPACE).withName(KafkaResources.kafkaStatefulSetName(CLUSTER_NAME)).get();
-                kafkaSts.setStatus(new StatefulSetStatus());
-
-                assertThat(kafkaSts, is(notNullValue()));
-                assertThat(kafkaSts.getSpec().getTemplate().getMetadata().getAnnotations(), hasEntry(StatefulSetOperator.ANNO_STRIMZI_IO_GENERATION, "0"));
-                assertThat(kafkaSts.getSpec().getTemplate().getMetadata().getAnnotations(), hasEntry(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_CERT_GENERATION, "0"));
-                assertThat(kafkaSts.getSpec().getTemplate().getMetadata().getAnnotations(), hasEntry(Ca.ANNO_STRIMZI_IO_CLIENTS_CA_CERT_GENERATION, "0"));
-
-                StatefulSet zkSts = client.apps().statefulSets().inNamespace(NAMESPACE).withName(KafkaResources.zookeeperStatefulSetName(CLUSTER_NAME)).get();
-                assertThat(zkSts, is(notNullValue()));
-                assertThat(zkSts.getSpec().getTemplate().getMetadata().getAnnotations(), hasEntry(StatefulSetOperator.ANNO_STRIMZI_IO_GENERATION, "0"));
-                assertThat(zkSts.getSpec().getTemplate().getMetadata().getAnnotations(), hasEntry(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_CERT_GENERATION, "0"));
-
-                assertThat(client.configMaps().inNamespace(NAMESPACE).withName(KafkaResources.kafkaMetricsAndLogConfigMapName(CLUSTER_NAME)).get(), is(notNullValue()));
+                StrimziPodSet sps = supplier.strimziPodSetOperator.client().inNamespace(NAMESPACE).withName(CLUSTER_NAME + "-kafka").get();
```

Comment:
It uses the method for the StatefulSet name since they are the same. I personally would not add more methods at this point. But using the statefulset one here is a good point.